packages/swapper/src/swappers/ButterSwap/swapperApi/getTradeQuote.test.ts

```diff
@@ -75,7 +75,7 @@ describe('getTradeQuote', () => {
     expect(result.isOk()).toBe(true)
     const tradeQuote = result.unwrap()
     // Use the actual returned value for the expected rate
-    expect(tradeQuote[0].rate).toBe('2298.70840740740740740741')
+    expect(tradeQuote[0].rate).toBe('2296.409699')
```

Comment:
Now how precision is now correct as a result of using our conventional util. There is only 6 dp for USDC, so that rate was absolutely wrong.